aws-single-instance-resiliency/Chart.yaml

```diff
@@ -22,3 +22,22 @@ version: 41.1.0
 # follow Semantic Versioning. They should reflect the version the application is using.
 # It is recommended to use it with quotes.
 appVersion: "3.41.1"
+
+keywords:
+  - artifacts
+  - dependency
+  - management
+  - sonatype
+  - nexus
+  - repository
+  - quickstart
+  - ci
+  - repository-manager
+  - nexus3
+home: https://www.sonatype.com/nexus-repository-oss
+icon: https://sonatype.github.io/helm3-charts/NexusRepo_Vertical.svg
+sources:
+  - https://github.com/sonatype/nexus-public
+maintainers:
+  - email:  support@sonatype.com
```

Comment:
This seems wrong given its only for Pro customers

Reply:
https://github.com/sonatype/nxrm3-helm-repository/pull/9/commits/3556fb6853d1b11ac43793d1b32dbb39a2594ae7

Reply:
Although I wonder if it should just be omitted since nexus-internal is private: https://github.com/sonatype/nxrm3-helm-repository/pull/9/commits/4eecf509cca80e619b8cf77ff7fb6b2722efe093

Reply:
I think you missed the point - a reference to the internal git repo isn't useful for customers.

My comment was that the email provided here is for creating support tickets.

Reply:
providing at least one email address may be required, but i'm not positive. you can try defining a chart without it.

Reply:
It says email address is optional [here](https://helm.sh/docs/topics/charts/), so will remove it.